titus_isolate/allocate/fall_back_cpu_allocator.py

```diff
@@ -24,24 +24,24 @@ def __init__(self, primary_cpu_allocator, secondary_cpu_allocator):
             self.__primary_allocator.__class__.__name__,
             self.__secondary_allocator.__class__.__name__))
 
-    def assign_threads(self, cpu, workload):
+    def assign_threads(self, cpu, workload_id, workloads):
         try:
-            cpu = self.__primary_allocator.assign_threads(cpu, workload)
+            cpu = self.__primary_allocator.assign_threads(cpu, workload_id, workloads)
             self.__primary_allocator_calls_count += 1
```

Comment:
What's this counter used for?

Reply:
Good catch.  We should be reporting a metric.